src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (Ball ball : balls) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball.toString());
```

Comment:
The enhanced for loop is not suitable for initializing the elements of the array. It iterates over the existing elements, but the assignment inside the loop does not change the actual array elements. Instead, use a standard for loop to assign new Ball objects to the array elements.